crates/bevy_gltf/src/loader.rs

```diff
@@ -390,7 +390,7 @@ async fn load_gltf<'a, 'b, 'c>(
             let primitive_label = primitive_label(&gltf_mesh, &primitive);
             let primitive_topology = get_primitive_topology(primitive.mode())?;
 
-            let mut mesh = Mesh::new(primitive_topology);
+            let mut mesh = Mesh::new(primitive_topology, false);
```

Comment:
can we make this a flag in the GltfLoaderSettings (or make settings.load_meshes into an enum maybe)?

Reply:
I'd rather not, I think. Users should instead find the specific meshes/textures after loading, and set those ones specifically to Keep.